apps/user-office-frontend/src/components/user/PeopleTable.tsx

```diff
@@ -292,12 +305,12 @@ const PeopleTable: React.FC<PeopleTableProps> = (props) => {
       // first clear the current page because if any row was unselected
       // the (un)select all option will select every rows
       // which would result in duplicates
-      setSelectedParticipants((selectedParticipants) =>
+      setSelectedParticipants?.((selectedParticipants) =>
```

Comment:
I wonder would it cause setSelectedParticipants(undefined=>undefined) or just skips it - refer to question 1

Reply:
I think it shouldn't happen because if `setSelectedParticipants` is not undefined the `selectedParticipants` is also an array.